development/applications/hedwig/src/flight_control.c

```diff
@@ -298,6 +300,26 @@ static void flight_control_reset(void)
   }  
 }
 
+void flight_control_calibrate_sensors(void)
+{
+  sensors_calibrate_cmd = true;
+}
+
+static uint8_t calibrate_sensors(void)
+{
+  fe_flight_mode_t prevMode = flight_control_mode;
+  if (flight_control_set_mode(FE_FLIGHT_MODE_CALIBRATING)) {
+    sensors_calibrate();
+    rtos_delay_ms(3000);
+    flight_control_set_mode(prevMode);
+    flight_control_reset();
+    return 1;
+  }
+  else {
+    return 0;
+  }
```

Comment:
i like not adding an else here, and just adding `return value;` outside of the if